### DISCUSSION TITLE
Recommendation for ceiling mount

### DISCUSSION THREAD

**Discussion by jbms** [TheRealWaldo/thermal#177] [Q&A]
I'm interested in using the MLX90640 for occupancy detection but I'm looking for some recommendations for mounting it on a ceiling in a way that allows it to be adjusted to point in the desired direction and maintains a reasonably discrete, non-ugly appearance.

**Comment by TheRealWaldo** [comment]
I'd say this depends entirely on the carrier board you chose, and the physical layout of the resulting board you build to host it.

I personally do not own the sensor you're using, so have no suggestions off the top of my head

Perhaps visit thingverse or sites like it for 3d printable models that others may have made?

**Comment by jbms** [comment]
Thanks for your reply.  I was thinking of using differential i2c (https://learn.sparkfun.com/tutorials/qwiic-differential-i2c-bus-extender-pca9615-hookup-guide/all) so that there isn't much required at the sensor itself.

I assume all of these thermal sensors are fairly similar in form factor --- do you have any suggestions for any others?

Another thought I had was to install the sensor in an electrical box installed in the ceiling, with a face plate made of some material that is mostly opaque to visible light but is transparent to the relevant far IR wavelength.  I think that could be done without occluding the fov and would allow me to position the sensor fairly easily, possibly just with tape.  Do you have any thoughts about the feasibility of that?

**Comment by TheRealWaldo** [comment]
I'm using 3D printed cases from STLs I found on Thingiverse for my AMG8833 sensors.  They're about an inch square and only expose the front of the sensor, which is relatively small.  https://www.thingiverse.com/thing:2836821

I can tell you that the form factor varies quite a bit based on the intended use, brand, make, and model.  The AMG8833 carrier boards differ between makers as well.

For your differential I2C project, I can't tell if that's an RJ-12 or RJ-45 connector, but plenty of low-voltage boxes with small form factors would work with the corresponding cabling.  I would be willing to bet you could find plates with pre-existing IR glass (think IR remote extenders) or push-in keystones with IR glass.

Pretty sure the wavelength would be close to what is in motion sensors, so the whiter glass you see on those might work as well if blending with a white plate is needed.  So perhaps you could look at in-wall motion sensor plates too.

Either way, keep us informed of your progress.

**Comment by TheRealWaldo** [comment]
@jbms did you get anywhere with your project?

**Comment by jbms** [comment]
Unfortunately I didn't have time to explore that more yet, but hopefully will find some time to look into it more in the future.
